test/runtests.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 # USAGE: Run this file using `npm test` from repository root
 
-mocha test/*_test.js
+mocha .test/*_test.js
```

Comment:
We can safely delete this file i believe?  Its only purpose was to wrap the package.json script entry iirc, so now that you're calling into mocha directly, It's not important to exist.